test/Interpreter/EvalSpec.hs

```diff
@@ -41,9 +41,54 @@ spec = do
     testProgramm "1 < 3" (Bool True)
     testProgramm "3 <= 3" (Bool True)
     testProgramm "3 == 2" (Bool False)
+    testProgramm "9 /= 10" (Bool True)
+    testProgramm "4 /= 6" (Bool True)
+    testProgramm "True /= 1" (Bool False)
```

Comment:
When testing your changes and those changes are correct, tests should be green.
`testProgramm` currently asserts only positive cases (we'll add tests for negative cases soon https://github.com/yurtsiv/mooncake/issues/3). So do not test negative cases yet and also keep in mind, that there're two failing tests (as the functionality is not implemented yet) that I forgot to remove:

![image](https://user-images.githubusercontent.com/18295488/94998216-76c5f580-05b0-11eb-8197-93d3008136ff.png)

Leave them as-is, we'll handle that in this issue https://github.com/yurtsiv/mooncake/issues/9

Thanks!